R/MzTabReader.R

```diff
@@ -360,17 +400,20 @@ getMSMSScans = function(identified_only = FALSE)
     res$mass.deviations..da. =  gsub(",", ";", res$mass.deviations..da., fixed = TRUE)
   }
   
- 
+  if (!"dp.modification" %in% colnames(res)) {
+    res$dp.modification = TRUE
```

Comment:
why true? could this also be NULL? (what is used by MaxQuant in the .txt files?)